fvm/evm/stdlib/contract.cdc

```diff
@@ -21,27 +21,27 @@ contract EVM {
         // height or number of the block
         height: UInt64,
         // hash of the block
-        hash: String,
+        hash: [UInt8],
```

Comment:
Maybe [UInt8;32] for hash values?

Reply:
Good idea! Does this also apply to the other hashes changed in the PR? i.e.:
- `BlockExecuted.parentHash`
- `BlockExecuted.receiptRoot`
- `BlockExecuted.transactionHashes`
- `TransactionExecuted.hash`

Reply:
Yes, also all of those, I suggest skipping BlockExecuted.transactionHashes as I'm going to deprecate it in my PR.

Reply:
Changed in aa8761d77ce39c3c442a9903c3149a0356789966